pyaedt/q3d.py

```diff
@@ -368,6 +370,355 @@ def edit_sources(
 
         return True
 
+    def export_matrix_data(
+        self,
+        file_name,
+        problem_type=None,
+        variations=None,
+        setup_name=None,
+        sweep=None,
+        reduce_matrix=None,
+        r_unit=None,
+        l_unit=None,
+        c_unit=None,
+        g_unit=None,
+        freq=None,
+        freq_unit=None,
+        matrix_type=None,
+        export_AC_DC_res=None,
+        precision=None,
+        field_width=None,
+        use_sci_notation=None,
+        length_settings=None,
```

Comment:
```suggestion
        length_settings="Distributed",
```

Reply:
length_setting? Seems like it should be singular and not plural (settings).